### DISCUSSION TITLE
CoPilot prompts in my markdown doc comments is mostly annoying

### DISCUSSION THREAD

**Discussion by lovettchris** [community/community#34269] [Copilot Conversations]
I can turn off CoPilot for "markdown" but I would also love a way to turn off CoPilot for "markdown comments" in my language (Lean4) because I'm finding that the millions of prompts I'm getting are all wrong and it is becoming so annoying I have to turn off copilot for my entire language, which I don't want to have to do...

**Comment by iamcjwilliams** [comment]
@lovettchris Thank you, Our copilot team will investigate this!

**Comment by tilmann1** [comment]
@lovettchris Which Copilot extension are you using? Thanks!

**Comment by lovettchris** [comment]
![image](https://user-images.githubusercontent.com/18707114/195769499-e2452f1d-af69-4d32-bd7a-8f18e1f90c24.png)

Now I have to say, I just did a big project editing tons of files with CoPilot turned on and the problem didn't seem to happen very often.  So I don't know if things are improving in this area.  One thing that might help is some sort of debouncing when I'm trying really fast, don't try and popup up suggestions until I'm done or something.
